projects/plugins/jetpack/extensions/store/membership-products/resolvers.js

```diff
@@ -31,6 +31,8 @@ export const getProducts = (
 	const origin = getQueryArg( window.location.href, 'origin' );
 	const path = addQueryArgs( '/wpcom/v2/memberships/status', {
 		source: origin === 'https://wordpress.com' ? 'gutenberg-wpcom' : 'gutenberg',
+		type: 'all',
+		public: true,
```

Comment:
If going through with the suggestion in https://github.com/Automattic/jetpack/pull/23714#issuecomment-1085622255 to rename public, this will need updating too

Reply:
Done :)